### DISCUSSION TITLE
Train a network with different boundary conditions.

### DISCUSSION THREAD

**Discussion by oruen321** [lululxvi/deepxde#1359] [Q&A]
Hi, thanks for the program. 

I'm wondering if there is a way to use more inputs in addition to time and space. for for example a DIffusion problem. 
for example: dc/(dx)|x=0 = **jn**, where **jn** is a flux. Can I train a PINN such that **jn** is also an input of the PINN? A network for like a bundle of solutions.

Thank you

**Comment by lululxvi** [comment]
Use DeepONet.
